src/app/routes/cpsAsset/utils/getAdditionalPageData.test.js

```diff
@@ -29,11 +30,16 @@ describe('getAdditionalPageData', () => {
       'http://localhost/mundo/sty-secondary-column.json',
       secondaryColumnJson,
     );
+    fetchMock.mock(
```

Comment:
Can we have a test for a service that does not have recommendations? This way we can ensure the hasRecommendations logic is working as expected. We do not want to be making unnecessary data fetches.

Edit* I can see hasRecommendations is tested separately, it might still be good to have a test case here, one for a servicer with recommendations and one without